library/crates/Cargo.toml

```diff
@@ -7,18 +7,20 @@ path = "" # ignored by cargo raze
 
 [dependencies]
 antlr-rust = "0.2.0"
-rocksdb = "0.17.0"
 cxx = "1.0.59"
-futures = "0.3.21"
-
-# gRPC dependencies
```

Comment:
Do we want to add purpose-related comments?

Should this be generalised?

Reply:
I would say no. As Rust becomes our primary language, the list of dependencies will grow very large, and it will be cumbersome to add a comment to each dependency explaining why it is there.

If someone is curious about a specific dependency, the workflow will be:
- Use git tool to find the PR that introduced the dependency
- View the PR comments

Minimally, the PR comments should state what repo(s) the dependency was used in when it was added, so we can then hunt down that repo and determine if it's still in use / needed.